services/collaboration/pkg/command/root.go

```diff
@@ -0,0 +1,29 @@
+package command
+
+import (
+	"os"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/clihelper"
+	"github.com/owncloud/ocis/v2/services/collaboration/pkg/config"
+	"github.com/urfave/cli/v2"
+)
+
+// GetCommands provides all commands for this service
+func GetCommands(cfg *config.Config) cli.Commands {
+	return []*cli.Command{
+		Server(cfg),
+		//Health(cfg),
```

Comment:
from my understanding any service has to provide these two commands .... :shrug:

Reply:
It's in the todo list. It wasn't available in the original repo and didn't want to add too much new code.
If it isn't strictly required, I'd rather delay the implementation in order to keep the size of the PR not too big.